local tests failing, but succeed in travis

```
$ invoke tests --group=python
COVERAGE_PROCESS_START=/media/logan/External/Code_Dev/nbgrader/.coveragerc py.test --cov nbgrader --no-cov-on-fail -v -x -m "not formgrader and not nbextensions"
======================================================= test session starts ========================================================
platform linux -- Python 3.5.1, pytest-2.9.1, py-1.4.31, pluggy-0.3.1 -- /home/logan/.virtualenvs/nbgrader-py35/bin/python3.5
cachedir: .cache
rootdir: /media/logan/External/Code_Dev/nbgrader, inifile: 
plugins: cov-2.2.1
collected 494 items 
.
.
.
nbgrader/tests/apps/test_nbgrader.py::TestNbGrader::test_check_version FAILED

============================================================= FAILURES =============================================================
_________________________________________________ TestNbGrader.test_check_version __________________________________________________

self = <nbgrader.tests.apps.test_nbgrader.TestNbGrader object at 0x7ff06e067898>
capfd = <_pytest.capture.CaptureFixture object at 0x7ff06e04f320>

    def test_check_version(self, capfd):
        """Is the version the same regardless of how we run nbgrader?"""
        if sys.platform == 'win32':
            out1 = "\r\n".join(run_command(["nbgrader.cmd", "--version"]).split("\r\n")[2:])
        else:
>           out1 = run_command(["nbgrader", "--version"])

/media/logan/External/Code_Dev/nbgrader/nbgrader/tests/apps/test_nbgrader.py:36: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/media/logan/External/Code_Dev/nbgrader/nbgrader/tests/__init__.py:124: in run_command
    copy_coverage_files()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def copy_coverage_files():
        root = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
        if os.getcwd() != root:
            coverage_files = glob.glob(".coverage.*")
            if len(coverage_files) == 0 and 'COVERAGE_PROCESS_START' in os.environ:
>               raise RuntimeError("No coverage files produced")
E               RuntimeError: No coverage files produced

/media/logan/External/Code_Dev/nbgrader/nbgrader/tests/__init__.py:107: RuntimeError
------------------------------------------------------- Captured stdout call -------------------------------------------------------
Python version 3.5.1 (default, Dec 18 2015, 00:00:00) 
[GCC 4.8.4]
nbgrader version 0.3.0.dev

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================ 122 tests deselected by "-m 'not formgrader and not nbextensions'" ================================
================================= 1 failed, 104 passed, 1 skipped, 122 deselected in 11.28 seconds =================================
```

This was run on a clean virtualenv on the master branch without any changes.
Am I doing something wrong?

Running `py.test` directly on `nbgrader/tests/apps/`:

```
$ py.test -x -vv --cov nbgrader nbgrader/tests/apps/
========================================================================================================================== test session starts ==========================================================================================================================
platform linux -- Python 3.5.1, pytest-2.9.1, py-1.4.31, pluggy-0.3.1 -- /home/logan/.virtualenvs/nbgrader-py35/bin/python3.5
cachedir: .cache
rootdir: /media/logan/External/Code_Dev/nbgrader, inifile: 
plugins: cov-2.2.1
collected 102 items 
.
.
.
nbgrader/tests/apps/test_nbgrader.py::TestNbGrader::test_check_version PASSED
.
.
.
nbgrader/tests/apps/test_nbgrader_submit.py::TestNbGraderSubmit::test_submit FAILED

=============================================================================================================================== FAILURES ================================================================================================================================
____________________________________________________________________________________________________________________ TestNbGraderSubmit.test_submit _____________________________________________________________________________________________________________________

self = <nbgrader.tests.apps.test_nbgrader_submit.TestNbGraderSubmit object at 0x7f2b9f939828>, exchange = '/tmp/tmpttcirnu7', cache = '/tmp/tmpo0ygn6y5', course_dir = '/tmp/tmpjq63dfpt'

    def test_submit(self, exchange, cache, course_dir):
        self._release_and_fetch("ps1", exchange, cache, course_dir)
        now = datetime.datetime.now()
    
        time.sleep(1)
        self._submit("ps1", exchange, cache)
    
        filename, = os.listdir(join(exchange, "abc101", "inbound"))
        username, assignment, timestamp1 = filename.split("+")
        assert username == os.environ["USER"]
        assert assignment == "ps1"
>       assert parse_utc(timestamp1) > now
E       assert datetime.datetime(2016, 4, 30, 13, 7, 24) > datetime.datetime(2016, 4, 30, 15, 7, 23, 47171)
E        +  where datetime.datetime(2016, 4, 30, 13, 7, 24) = parse_utc('2016-04-30 13:07:24 UTC')

/media/logan/External/Code_Dev/nbgrader/nbgrader/tests/apps/test_nbgrader_submit.py:70: AssertionError
------------------------------------------------------------------------------------------------------------------------- Captured stderr call --------------------------------------------------------------------------------------------------------------------------
[ReleaseApp | INFO] Source: /tmp/tmpjq63dfpt/release/./ps1
[ReleaseApp | INFO] Destination: /tmp/tmpttcirnu7/abc101/outbound/ps1
[ReleaseApp | INFO] Released as: abc101 ps1
[FetchApp | INFO] Source: /tmp/tmpttcirnu7/abc101/outbound/ps1
[FetchApp | INFO] Destination: /tmp/tmp84n05cwt/ps1
[FetchApp | INFO] Fetched as: abc101 ps1
[SubmitApp | INFO] Source: /tmp/tmp84n05cwt/ps1
[SubmitApp | INFO] Destination: /tmp/tmpttcirnu7/abc101/inbound/logan+ps1+2016-04-30 13:07:24 UTC
[SubmitApp | INFO] Submitted as: abc101 ps1 2016-04-30 13:07:24 UTC
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================================================================= 1 failed, 84 passed in 242.09 seconds =================================================================================================================
```

--- a/nbgrader/tests/__init__.py
+++ b/nbgrader/tests/__init__.py
@@ -4,6 +4,7 @@ import shutil
 import subprocess as sp
 import sys
 import logging
+import warnings
 
 from six import StringIO
 from nbformat.v4 import new_code_cell, new_markdown_cell
@@ -104,7 +105,7 @@ def copy_coverage_files():
     if os.getcwd() != root:
         coverage_files = glob.glob(".coverage.*")
         if len(coverage_files) == 0 and 'COVERAGE_PROCESS_START' in os.environ:
-            raise RuntimeError("No coverage files produced")
+            warnings.warn("No coverage files produced")
         for filename in coverage_files:
             shutil.copyfile(filename, os.path.join(root, filename))
 

